lib/vector-api-client/graphql/schema.json

```diff
@@ -472,6 +472,92 @@
           "name": "ComponentProcessedEventsThroughput",
           "possibleTypes": null
         },
+        {
+          "description": null,
+          "enumValues": null,
+          "fields": [
+            {
+              "args": [],
+              "deprecationReason": null,
+              "description": "Component name",
+              "isDeprecated": false,
+              "name": "name",
+              "type": {
+                "kind": "NON_NULL",
+                "name": null,
+                "ofType": {
+                  "kind": "SCALAR",
+                  "name": "String",
+                  "ofType": null
+                }
+              }
+            },
+            {
+              "args": [],
+              "deprecationReason": null,
+              "description": "Events inputted throughput",
```

Comment:
Just a reminder here that if the comments changes are accepted, a re-run of `make build-graphql-schema` will keep these in sync.